_pages/mobile-interview-project.md

```diff
@@ -8,40 +8,55 @@ header:
 
 # Hello!
 
-*This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: An Employee Directory App. You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process please share them with your recruiting contact and we’ll get back to you ASAP. Good luck!*
+Thank you for choosing to interview at Square! We are so excited for you and we hope that you have a great experience with us.
+
+This site outlines the info and guidelines you’ll need to complete Square’s Mobile Take-Home Project: an employee directory app.
+
+You’ll have a week to complete the question – the actual engineering work should take about 3-4 hours, and you can split this across the week however you see fit (all at once, over a few days, etc). If you have any questions during the process, please share them with your recruiting contact and we’ll get back to you ASAP.
+
+Good luck!
 
 # Overview
-Build an employee directory app that shows a list of employees parsed from the provided JSON endpoint.
 
-Take care to properly handle errors returned from the endpoint (or other network errors like timeouts), and ensure you do not waste network bandwidth – load expensive resources such as large photos on-demand only. Photos at a given URL will never change. Once one is loaded, you do not need to reload the photo. If an employee’s photo changes, they will be given a new photo URL (eg, you can treat the URL as a GUID).
+Build an employee directory app that shows a list of employees from the provided endpoint.
+
+The app should display a list (or any kind of collection view!) which shows all the employees returned from the JSON endpoint described below. Each item in the view should contain a summary of the employee, including their photo, name, and team at minimum. You may add more information to the summary if you want, or sort employees in any fashion you’d like – sort and group by name, team, etc.
```

Comment:
nit: Does "collection view" apply to iOS as well?   I wonder if we should just keep it as a List or say List or Grid.  

Examples:
```
The app should display a list which shows all the employees...
```

```
The app should display a list (or grid if you prefer!) which shows all the employees
```

Maybe iOS uses the term "collection view" too though and this is irrelevant.  Take with a grain of salt!

Reply:
Yes, collection view is a common iOS term: https://developer.apple.com/documentation/uikit/uicollectionview